src/shared/utils.js

```diff
@@ -19,6 +19,10 @@ export const Utils = {
     return currentTab[0];
   },
 
+  isSupportedOs(os) {
+    return ["win"].includes(os);
```

Comment:
linux tooooo

Reply:
I think QA has nixed shipping the MVP on Linux as they've only been testing on Windows so far.